### ISSUE TITLE
Jobs Failing with error from docker daemon

### ISSUE THREAD

**Issue by amiryala2698** [NVIDIA/nvidia-docker#1047]
error thrown :

docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "process_linux.go:402: container init caused \"process_linux.go:385: running prestart hook 1 caused \\\"error running hook: exit status 1, stdout: , stderr: exec command: [/usr/bin/nvidia-container-cli --load-kmods configure --ldconfig=@/sbin/ldconfig.real --device=1 --compute --utility --video --graphics --require=cuda>=10.0 brand=tesla,driver>=384,driver<385 --pid=59574 /var/lib/docker/120000.120000/overlay2/349b0d0f35037cccda45e6baf9151e8c0eb3a01174a7b11d64799d1107a1e9c3/merged]\\\\nnvidia-container-cli: initialization error: driver error: timed out\\\\n\\\"\"": unknown.

Once the docker daemon on a node throws this error, from then all the jobs that land on the node FAIL as soon as they start.

any jobs that were RUNNING before this error shows up seem to run fine.

 a reboot on the node usually fixes the issue, but I want to know the root cause of this error and see if there is anything we can do before the daemon on the host starts throwing this error.

There are no steps to reproduce this issue that we know of, but you can work on this node "172.22.1.199" (cosmos node) as it is throwing the error now. I have drained it for your work.

DOCKER VERSION:
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:17:20 2018
 OS/Arch:      linux/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:15:30 2018
  OS/Arch:      linux/amd64
  Experimental: false

**Comment by rajatchopra** [comment]
Can you run the following from the host:
`sudo nvidia-container-cli -k -d /dev/tty info`

**Comment by Ethyling** [comment]
Hey, can you check if the driver is still working (by running `nvidia-smi` for example) when the bug happen on your host?

**Comment by amiryala2698** [comment]
@rajatchopra 
172.22.1.199 | SUCCESS | rc=0 >>
NVRM version:   418.40.04
CUDA version:   10.1

Device Index:   0
Device Minor:   0
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-51ba062a-e4c6-3f5e-f722-7ddcfa4dea0f
Bus Location:   00000000:06:00.0
Architecture:   7.0

Device Index:   1
Device Minor:   1
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-b6f89044-85b9-fa44-2c98-3ea5e1389c3a
Bus Location:   00000000:07:00.0
Architecture:   7.0

Device Index:   2
Device Minor:   2
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-987c5738-bbf8-10a0-d74b-2a1e60adeadb
Bus Location:   00000000:0a:00.0
Architecture:   7.0

Device Index:   3
Device Minor:   3
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-0e407346-10b7-4ced-db02-35eca941c250
Bus Location:   00000000:0b:00.0
Architecture:   7.0

Device Index:   4
Device Minor:   4
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-c775ab0e-6816-7d88-efc7-98dbd0bf744f
Bus Location:   00000000:85:00.0
Architecture:   7.0

Device Index:   5
Device Minor:   5
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-c88f6d95-0970-f29f-edc2-5ef82a6ccc86
Bus Location:   00000000:86:00.0
Architecture:   7.0

Device Index:   6
Device Minor:   6
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-1bffe263-e67b-551b-0de2-6cde658bdfae
Bus Location:   00000000:89:00.0
Architecture:   7.0

Device Index:   7
Device Minor:   7
Model:          Tesla V100-SXM2-16GB
Brand:          Tesla
GPU UUID:       GPU-8bcdc7a0-ed99-e25f-c09c-ef067ddb0d95
Bus Location:   00000000:8a:00.0
Architecture:   7.0

**Comment by amiryala2698** [comment]
@Ethyling 
yes, I ran nvidia-smi and I see that the driver is working, But i observe that it takes some time to give out the output. does that indicate anything?

**Comment by Ethyling** [comment]
@amiryala2698 More than 10 seconds?

**Comment by amiryala2698** [comment]
@Ethyling , time nvidia-smi gives me

real	0m45.182s
user	0m0.008s
sys	0m12.420s

**Comment by Ethyling** [comment]
@amiryala2698 Is nvidia-persistenced running on the host?

**Comment by amiryala2698** [comment]
how do i check that?

**Comment by Ethyling** [comment]
`ps aux | grep persistenced` should do the work

**Comment by amiryala2698** [comment]
looks like it is running.
![image](https://user-images.githubusercontent.com/46169733/63452435-21a24100-c3fb-11e9-81dd-eeb0c357bf66.png)

**Comment by Ethyling** [comment]
Ok, could you give us the output of the `nvidia-smi` command?

**Comment by amiryala2698** [comment]
root@sc-sdgx-654:/home/lab# time nvidia-smi
Wed Aug 21 16:48:37 2019
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 418.40.04    Driver Version: 418.40.04    CUDA Version: 10.1     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Tesla V100-SXM2...  On   | 00000000:06:00.0 Off |                    0 |
| N/A   36C    P0    46W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   1  Tesla V100-SXM2...  On   | 00000000:07:00.0 Off |                    0 |
| N/A   36C    P0    41W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   2  Tesla V100-SXM2...  On   | 00000000:0A:00.0 Off |                    0 |
| N/A   36C    P0    42W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   3  Tesla V100-SXM2...  On   | 00000000:0B:00.0 Off |                    0 |
| N/A   35C    P0    42W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   4  Tesla V100-SXM2...  On   | 00000000:85:00.0 Off |                    0 |
| N/A   35C    P0    40W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   5  Tesla V100-SXM2...  On   | 00000000:86:00.0 Off |                    0 |
| N/A   35C    P0    40W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   6  Tesla V100-SXM2...  On   | 00000000:89:00.0 Off |                    0 |
| N/A   37C    P0    43W / 300W |      0MiB / 16130MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   7  Tesla V100-SXM2...  On   | 00000000:8A:00.0 Off |                    0 |
| N/A   39C    P0   ERR! / 300W |      0MiB / 16130MiB |     87%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+

real	0m45.182s
user	0m0.008s
sys	0m12.420s
root@sc-sdgx-654:/home/lab# exit
